cypress/support/pages/home.pageObject.js

```diff
@@ -3,8 +3,23 @@ import PageObject from '../PageObject';
 class homePageObject extends PageObject {
   url = '/#/';
 
-  get usernameLink() {
-    return cy.getByDataCy('/profile/');
+  usernameLink(username) {
+    cy.getByDataCy(`/profile/${username}`)
```

Comment:
Use a more descriptive name for the `usernameLink` method, like `getUsernameLink` or `findUsernameLink`.